### ISSUE TITLE
Phpstorm v2017.3 : Unknown CSS at-rule - Not using .stylelintrc file

### ISSUE THREAD

**Issue by TheFrankman** [Jardinero/stylelint-plugin#14]
This is my .stylelintrc which is sat in the project root

```
{
  "extends": "stylelint-config-recommended",
  "rules": {
    "at-rule-no-unknown": [
      true,
      {
        "ignoreAtRules": [
          "extend",
          "extends",
          "tailwind"
        ]
      }
    ],
    "block-no-empty": null
  }
}
```

In my css the following is getting highlighted as an error by stylelint `@tailwind utilities;`

The error is `Unknown CSS at-rule`

This should be being ignored by stylelint. 

My configuration is 

Node Interpreter : `/user/local/bin/node`
Stylelint package : `/usr/local/lib/node_modules/stylelint`

Am i missing some configuration that tells the IDE  / stylelint where to locate the .stylelintrc

**Comment by TheFrankman** [comment]
I have upgraded PHPstorm to v2018.1 and the issue persists.

**Comment by niutech** [comment]
PHPStorm supports Stylelint [natively](https://www.jetbrains.com/help/phpstorm/using-stylelint-code-quality-tool.html), have you tried it?

**Comment by TheFrankman** [comment]
@niutech Yes I have tried using File Watchers, even using the documentation you linked. It had the same issue which is why I came to use this repository.

**Comment by terryupton** [comment]
@thefrankman did you solve this? I’m trying to setup and have the same issue.

**Comment by adrienrobertskoove** [comment]
Still having this problem, any way to add @ rules manualy or change the linter ?

**Comment by terryupton** [comment]
Anyone managed to solve this at all? I have just come back to see if I can get this working again.
cc @TheFrankman @adrienrobertskoove

**Comment by terryupton** [comment]
For anyone else, I believe this is down to PHPStorm running it's own inspection rules.
It should be as straight forward as disabling these warnings in the PHPstorm Settings > Editor > Inspection

See below:
<img width="1388" alt="Screenshot 2020-02-18 at 12 24 41" src="https://user-images.githubusercontent.com/5698596/74735999-bde1bf00-5249-11ea-895a-a6806a6b0869.png">

unticking the 'Unknown CSS at-rule' has removed these warnings/highlights.

**Comment by adrienrobertskoove** [comment]
@terryupton No so far I just left them there. I'm afraid of disabling the whole @ rules as there's more than one and might pass on real errors.
`@import` is not trigerring the linter but `@screen`, `@apply` does

**Comment by MvnTruong** [comment]
You can suppress this on a per-line basis with `/*noinspection CssInvalidAtRule*/`, so the top of your Tailwind's `main.css` file would look like this:

```
/*noinspection CssInvalidAtRule*/
@tailwind base;

/*noinspection CssInvalidAtRule*/
@tailwind components;

/*noinspection CssInvalidAtRule*/
@tailwind utilities;
```
